package.json

```diff
@@ -23,32 +23,30 @@
   "scripts": {
     "lint": "npx eslint source/js --ext .js",
     "test": "jest source/__tests__",
-    "generate-docs": "node_modules/.bin/jsdoc --configure .jsdoc.json --verbose",
     "coverage": "jest --coverage",
+    "generate-docs": "node_modules/.bin/jsdoc --configure .jsdoc.json --verbose",
+    "copy-docs": "mkdir dist && mkdir dist/docs/ && cp -r source/docs/* dist/docs",
```

Comment:
Should we move these command to the workflow instead of making them a script? I don't think we're ever using it outside of the workflow

Reply:
Yeah I'll remove them.